Makefile

```diff
@@ -201,11 +201,11 @@ golangci-lint:
 
 test:
 ifndef WHAT
-	$(Q)$(GO_CMD) test -race -coverprofile=coverage.txt -covermode=atomic \
+	$(Q)$(GO_CMD) test -tags testing -race -coverprofile=coverage.txt -covermode=atomic \
```

Comment:
this could be an override that travis/any CI sets if it needs to skip avx but is run by default (or vice versa)?

Reply:
Ok. Changed it into an overrideable set of build tags specific to building tests. By default those tags are set to disable the registration of the AVX collector.